input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md

```diff
@@ -0,0 +1,8 @@
+### Usage scenarios
+The following are supported usage scenarios for this profile:
+
+- Query for a request for a diagnostic service for a patient
```

Comment:
Take out usage scenarios until the interaction and obligation meetings

Reply:
Done

Reply:
Remove it entirely, not just placeholder.

Reply:
Done